### DISCUSSION TITLE
Has badRequest been removed from remix-utils v2?

### DISCUSSION THREAD

**Discussion by dmwallace** [sergiodxa/remix-utils#231] [Q&A] [answered]
Has badRequest been removed from remix-utils v2?

**Comment by sergiodxa** [comment, marked as answer]
I imagine you mean in v7, yes, those functions which just wrap `json` to add a status code are going to be removed, the response helpers that will remain are the `redirectBack`, `notModified` (because it has no body), and the ones for different content types, and other ones like `jsonHash` since they add more features over just setting a status code.
